crates/bindings-sys/Cargo.toml

```diff
@@ -11,3 +11,4 @@ bench = false
 
 [dependencies]
 getrandom = {workspace = true, optional = true}
+spacetimedb-primitives = { path = "../primitives" }
```

Comment:
Again for my edification: Why does this one not need to name the 0.7.1 tag? Is it because bindings-sys isn't published or something?

Reply:
I left it as was before. I also wonder if we require the tag.